src/NHSD.BuyingCatalogue.Solutions.Application/Domain/Epics/Epic.cs

```diff
@@ -0,0 +1,11 @@
+using System;
```

Comment:
Need to review this class. The domain of an epic should look similar to the epic data. Can't remember what this class is trying to represent.